docs/reference/errors-and-warnings/NU3043.md

```diff
@@ -0,0 +1,43 @@
+---
+title: NuGet Warning NU3043
+description: NU3043 warning code
+author: kartheekp-ms
+ms.date: 09/11/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU3043"
+---
+
+# NuGet Warning NU3043
+
+> Invalid value for `--certificate-fingerprint` option in the `dotnet nuget sign` command or the `CertificateFingerprint` option in the `NuGet.exe sign` command.
+The value must be a SHA-256, SHA-384, or SHA-512 certificate fingerprint (in hexadecimal).
+
+> This warning will be promoted to an error around the `.NET 10` timeframe.
+
+## Issue
+
+Starting with `.NET 9 Preview 7` and `NuGet.exe 6.12`, `NU3043` warning is raised when a SHA-1 certificate fingerprint is passed to the sign commands.
```

Comment:
the exact preview of .NET 9 isn't important. New previews/rcs come quickly, and soon enough .NET 9 will be GA.

And like the previous comment, .NET 9 and NuGet.exe aren't code snippets, so putting them in code blocks doesn't feel right to me. The NU code does feel fine with me, but I'd also be fine with it being taken out a code block.

Reply:
Fixed in the latest commit.